src/components/Integrations/BigQuery/ControlPanel/ControlPanel.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { useState, useMemo, useCallback } from 'react';
+import { useIntl } from 'react-intl';
+import MultiInputContainer from './MultiInputContainer';
+import { FormattedMessageMarkdown } from '../../../../i18n/FormattedMessageMarkdown';
+import { VerticalLine } from './style/styles.js';
+import datastudiogif from '../../../../img/google-data-studio.gif';
+
+interface IControlPanelProps {
+  values?: string[];
+  onChange: (newValues: string[]) => void;
+}
+
+const ControlPanel: React.FunctionComponent<IControlPanelProps> = (props) => {
+  const intl = useIntl();
+  const i18 = useCallback(
+    (id: string, values?: any) => {
+      return intl.formatMessage({ id: id }, values);
+    },
+    [intl],
+  );
+
+  const [values, setValues] = useState<string[]>([]);
```

Comment:
What are these values? emails?

In that case, what do you think about renaming it?

```suggestion
  const [emails, setEmails] = useState<string[]>([]);
```